docs/guide/data_types/maps.rst

```diff
@@ -1,203 +1,61 @@
-****
-Maps
-****
-
-Maps in sunpy are are 2-dimensional data associated with a coordinate system. In
-this guide, we will cover some of the basic functionality of maps. Once you've
-read through this guide check out :doc:`/code_ref/map` for a more thorough look
-at sunpy maps. There you can see what instruments are currently supported or you
-can access the code reference for each instrument-specific map subclass.
-
-Creating maps
-=============
-To make things easy, sunpy can download several example files which are used
-throughout the docs. These files have names like
-``sunpy.data.sample.AIA_171_IMAGE`` and ``sunpy.data.sample.RHESSI_IMAGE``. To
-create a `~sunpy.map.Map` from the the sample AIA image
-type the following into your Python shell::
+*********
+Map Guide
+*********
 
-    >>> import sunpy
-    >>> import sunpy.map
-    >>> import sunpy.data.sample  # doctest: +REMOTE_DATA
+Map objects in **sunpy** are two-dimensional data associated with a coordinate system.
+This class offers many advantages over using packages such as `astropy.io.fits` to open 2D solar data in Python.
+**sunpy** Maps recognize data from 19 sources (see :doc:`/code_ref/map` for a complete list) and can provide instrument-specific information when plotting and even correct or add metadata when it is missing from the original file.
```

Comment:
```suggestion
**sunpy** Maps can be used with any image with two spatial axes that has FITS standard compliant metadata.
There are also 19 sources (see :doc:`/code_ref/map` for a complete list) that provide instrument-specific information when plotting and in some cases interpret missing metadata when it is missing from the original file.
```
I we should be clear that it's not only maps with map sources that can be opened? Also we deliberately don't add any missing metadta, just guess it 😃